tests/test_tron_tools.py

```diff
@@ -187,6 +189,9 @@ def test_get_env(self, action_config, executor, monkeypatch):
             else:
                 assert not any([env.get("SPARK_OPTS"), env.get("CLUSTERMAN_RESOURCES")])
 
+        # ensure soa-configs SHA is not acquired to add as env var
+        assert mock_read_soa_metadata.call_count == 0
```

Comment:
would the intent here be clearer without a comment if we did something like

```
assert "PAASTA_SOA_CONFIGS_SHA" not in env
```

Reply:
Yeah makes sense.